### PR TITLE
feature/topics service

### PR DESCRIPTION
**Pull request by kgajowy**
base branch: `chore/hooks`

Added:
* Topics Container (handling logic of getting data from service/store)
* allow eslint to understand jasmine's keywords
* reorganized structure (moved Topic's related components to `src/topics/components/*`
* improved TopicsList (prop types, tests)
* added TopicModel & Service

### CODE REVIEW

File: src/shared/models/__tests__/TopicModel.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import TopicModel from '../TopicModel';
+
+describe('TopicModel', () => {
+    
+    describe('objet creation from backend data', () => {
```
Comment (Tuhaj) [inline src/shared/models/__tests__/TopicModel.test.js]: Last time I saw this typo in 2015. It was made by a French developer as `objet` is an object in French. Do you know French?

File: src/shared/services/TopicService.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        TopicModel.fromBackendData({ title: 'React + Redux', describe: 'Lorem ipsum' }),
+        TopicModel.fromBackendData({ title: 'React + Redux + Thunk', describe: 'Lorem ipsum' }),
+    ];
+    
+    static async getSubmittedTopics() {
+        return this.topics;
+    }
+    
+    static async getPastTopics() {
+        return this.topics;
```
Comment (Tuhaj) [inline src/shared/services/TopicService.js]: the same as `getSubmittedTopics`? Are you over engineering? 🙃

File: src/topic/TopicContainer.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import TopicService from '../shared/services/TopicService';
+
+class TopicContainer extends React.Component {
+    
+    state = {
+        topics: []
+    };
+    
+    async componentDidMount() {
+        //TODO in next PR: convert to Redux and add tests
```
Comment (Tuhaj) [inline src/topic/TopicContainer.js]: Reservation for this TODO!

File: src/shared/services/TopicService.js (hunk 2 above)
Comment (kgajowy) [reply src/shared/services/TopicService.js]: @Tuhaj on design, we have `submitted topics` and some kind of `past workshops` which are quite a different things. There I put a different methods just for a sake of presence for future reference. Until    we actually have backend/firebase, it is kind of a dummy implementation. What is your take there?

File: src/shared/models/__tests__/TopicModel.test.js (hunk 1 above)
Comment (kgajowy) [reply src/shared/models/__tests__/TopicModel.test.js]: No :( 
fixed in [https://github.com/warsawjs/topics-manager/pull/8/commits/47823e3cd8ce749d9c0b7260a9eefd564b877fd2]

**Comment by kgajowy** [comment]
@Tuhaj please re-approve

File: src/shared/services/TopicService.js (hunk 2 above)
Comment (Tuhaj) [reply src/shared/services/TopicService.js]: I get it, it's ok

**Comment by Tuhaj** [comment]
@kgajowy done!

**Pull request merged**